docs/spec/data-plane.md

```diff
@@ -91,6 +91,14 @@ delivered.
 CloudEvents received by Sink MAY have
 [Distributed Tracing Extension Attribute](https://github.com/cloudevents/spec/blob/v1.0/extensions/distributed-tracing.md).
 
+### Event reply header
+
+An event sender, including Source and Broker and Channel, SHOULD include
+the event reply header `K-Eventing-Event-Reply: response` in an event delivery
+request if it supports response events. If no event reply header is present
+or the header is present but does not include the value `response`,
+the Sink SHOULD assume that response events are NOT supported.
```

Comment:
a single `NOT` is not spec language.

Reply:
I think this could be reworded to be RFC compliant as: "the Sink SHOULD NOT assume that response events are supported."

Reply:
Changed to lower case.